templates/web/oxfordshire/around/_report_banner.html

```diff
@@ -0,0 +1,12 @@
+<h1 class="big-green-banner">
+    <!-- The mobile interface uses this element to construct
+         the "Place pin on map" message at the top of the screen.
+         But we don’t want it to show up on desktop. -->
+</h1>
```

Comment:
The contents of this `h1` are used if JavaScript is not available in the mobile interface. So it should not be empty. Presumably it is being hidden by the CSS on the desktop anyway?